src/xpcc/architecture/platform/driver/core/cortex/driver.xml

```diff
@@ -27,17 +27,17 @@
 		<parameter name="linkerscript_table_heap_extern" type="string"></parameter>
 
 		<!-- lpc -->
-		<template device-platform="lpc" out="linkerscript.ld">stm32/stm32_ram.ld.in</template>
+		<template device-platform="lpc" out="linkerscript.ld">linkerscript/stm32_ram.ld.in</template>
```

Comment:
Our one LPC device doesn't really count, it's just coincidence that this linkerscript fits the bill.
Until we have a better overview over other vendor's memory maps, I want to keep it stm32 focussed.